services/app-api/utils/kafka/kafka-source-lib.ts

```diff
@@ -54,7 +54,7 @@ class KafkaSourceLib {
       throw new Error("Missing Broker Config. ");
     }
     // Setup vars
-    this.stage = process.env.STAGE ? process.env.STAGE : "";
+    this.stage = process.env.stage ? process.env.stage : "";
```

Comment:
```suggestion
    this.stage = process.env.stage;
```
Can we be sure there will be a stage?